hermes/src/ontology/mod.rs

```diff
@@ -78,7 +77,45 @@ where
     D: serde::Deserializer<'de>,
 {
     use serde::de::Error;
-    use serde::Deserialize;
     String::deserialize(deserializer)
         .and_then(|string| base64::decode(&string).map_err(|err| Error::custom(err.to_string())))
 }
+
+#[derive(Debug, Clone, PartialEq, PartialOrd, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct LoadedForSiteMessage {
+    /// The site concerned
+    pub site_id: String,
+}
+
+impl<'de> HermesMessage<'de> for LoadedForSiteMessage {}
+
+#[derive(Debug, Clone, PartialEq, PartialOrd, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ReloadRequestMessage {
+    /// Id associated to a reload request operation of a component
+    pub id: String,
+}
+
+impl<'de> HermesMessage<'de> for ReloadRequestMessage {}
+
+#[derive(Debug, Clone, PartialEq, PartialOrd, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ReloadedMessage {
+    /// Id associated to a reload operation of a component
+    pub id: String,
+}
+
+impl<'de> HermesMessage<'de> for ReloadedMessage {}
+
+impl From<ReloadRequestMessage> for ReloadedMessage {
```

Comment:
remove these ?

Reply:
I thought this marker trait was needed for every hermes message ?